src/routes/safe/components/Transactions/TxsTable/ExpandedTx/index.jsx

```diff
@@ -41,8 +41,8 @@ const ExpandedTx = ({ cancelTx, tx }: Props) => {
   const [openModal, setOpenModal] = useState<OpenModal>(null)
   const openApproveModal = () => setOpenModal('approveTx')
   const closeModal = () => setOpenModal(null)
-  const thresholdReached = !INCOMING_TX_TYPES.includes(tx.type) && threshold <= tx.confirmations.size
-  const canExecute = !INCOMING_TX_TYPES.includes(tx.type) && nonce === tx.nonce
+  const thresholdReached = !INCOMING_TX_TYPES[tx.type] && threshold <= tx.confirmations.size
```

Comment:
```
const isIncomingTx = !!INCOMING_TX_TYPES[tx.type]
```
wdyt?

Reply:
this can replace multiple occurencies of `!INCOMING_TX_TYPES[tx.type]`

Reply:
perfect! https://github.com/gnosis/safe-react/pull/812/commits/4653243ac8aa76c63920d91d4baf9e5167d33833